gh-publish.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+set -euo pipefail
+
+git_branch=${GITHUB_REF##*/}
+git_hash=$(git rev-parse --short "$GITHUB_SHA")
+artifact_name="roar-extension $git_branch - $GITHUB_RUN_NUMBER @$git_hash"
+
+PUBLISH_DIR=publish
+
+ARTIFACT_FILE_NAME="artifacts/$artifact_name.zip"
```

Comment:
Spaces in file names can lead to problems in the future and for anyone that downloads the artifact.